File: src/main/java/com/kapilkoju/nepse/data/floorsheet/FloorSheet.java
package com.kapilkoju.nepse.data.floorsheet;

import lombok.Builder;
import lombok.Value;

import java.math.BigDecimal;

@Value
@Builder
class FloorSheet {
    private Long contractNo;
    private String stockSymbol;
    private Integer buyerBroker;
    private Integer sellerBroker;
    private Integer quantity;
    private BigDecimal rate;
    private BigDecimal amount;
}

File: src/main/kotlin/com/kapilkoju/nepse/data/floorsheet/model/FloorSheet.kt
package com.kapilkoju.nepse.data.floorsheet.model

import java.math.BigDecimal
import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "floor_sheet")
data class FloorSheet(
        @Id
        val contractNo: Long,
        val stockSymbol: String,
        val buyerBroker: String,
        val sellerBroker: String,
        val quantity: Int,
        val rate: BigDecimal,
        val amount: BigDecimal
)